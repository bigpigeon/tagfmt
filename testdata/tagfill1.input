//gofmt -f "*"
package main

type User struct {
	Name     string `json:"name"`
	Password string `xml:"password"`
    EmptyTag string
	City     string `json:"group" xml:"group"`
	State    string `gorm:"type:varchar(64)" xml:"state"`
}