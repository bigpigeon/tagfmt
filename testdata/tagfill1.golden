//gofmt -f "*"
package main

type User struct {
	Name     string `json:"name"    xml:""`
	Password string `xml:"password" json:""`
	EmptyTag string
	City     string `json:"group"            xml:"group" gorm:""`
	State    string `gorm:"type:varchar(64)" xml:"state" json:""`
}
